besu/src/main/java/org/hyperledger/besu/cli/subcommands/PublicKeySubCommand.java

```diff
@@ -62,25 +63,20 @@
   private CommandSpec spec; // Picocli injects reference to command spec
 
   private final PrintStream out;
-  private final KeyLoader keyLoader;
+  private final Supplier<NodeKey> nodeKeySupplier;
```

Comment:
You can call this just `nodeKey` as the Supplier prefix can be inferred from the type.

Reply:
👍 suggestion applied.